client/activities/activity-button.tsx

```diff
@@ -116,25 +115,24 @@ export interface ActivityButtonProps {
    * A hotkey to register for the button. Pressing the specified modifiers and key will result in
    * the button being clicked programmatically.
    */
-  hotkey?: {
-    keyCode: number
-    altKey?: boolean
-    shiftKey?: boolean
-    ctrlKey?: boolean
-  }
+  hotkey: HotkeyProp
 }
 
 export const ActivityButton = React.memo(
-  React.forwardRef<HTMLButtonElement, ActivityButtonProps>(
-    ({ label, icon, disabled, glowing, count, onClick, hotkey }, ref) => {
-      const localRef = useRef<HTMLButtonElement>()
+  React.forwardRef(
+    (
+      { label, icon, disabled, glowing, count, onClick, hotkey }: ActivityButtonProps,
+      ref: React.ForwardedRef<HTMLButtonElement>,
+    ) => {
       const [buttonProps, rippleRef] = useButtonState({
         disabled,
         onClick,
       })
-      const setRefs = useCallback(
+
+      const buttonRef = useRef<HTMLButtonElement>()
+      const setButtonRef = useCallback(
         (elem: HTMLButtonElement | null) => {
-          localRef.current = elem !== null ? elem : undefined
+          buttonRef.current = elem !== null ? elem : undefined
```

Comment:
Think the safer thing to do now is probably just `elem ?? undefined` (this current logic is risky if this ref was passed into another multiplexing ref that also did this conversion)